arm-insights/2015-07-01/swagger/insightsManagementClient_DiagnosticsSettings.json

```diff
@@ -59,7 +59,7 @@
             "in": "body",
             "required": true,
             "schema": {
-              "$ref": "#/definitions/ServiceDiagnosticSettingsCreateOrUpdateParameters"
+              "$ref": "#/definitions/ServiceDiagnosticSettingsResource"
             },
             "description": "Parameters supplied to the operation."
           },
```

Comment:
- Is MetricSettings.retentionPolicy not a required property ?
- Is LogSettings.retentionPolicy not a required property ?
- What are the required property of "ServiceDiagnosticSettings" model ?
- Can you provide an example in the description "ServiceDiagnosticSettings.workspaceId"? This will let users know the format of the Id.